sway-fmt-v2/src/utils/language/punctuated.rs

```diff
@@ -18,62 +18,65 @@ where
         formatted_code: &mut FormattedCode,
         formatter: &mut Formatter,
     ) -> Result<(), FormatterError> {
-        match formatter.shape.code_line.line_style {
-            LineStyle::Normal => {
-                let value_pairs = &self.value_separator_pairs;
-                for (type_field, punctuation) in value_pairs.iter() {
-                    type_field.format(formatted_code, formatter)?;
-                    punctuation.format(formatted_code, formatter)?;
-                    write!(formatted_code, " ")?;
-                }
+        if self.value_separator_pairs.is_empty() && self.final_value_opt.is_none() {
+        } else {
```

Comment:
Rather than:
```rust
        if self.value_separator_pairs.is_empty() && self.final_value_opt.is_none() {
        } else {
```
Could be
```rust
        if !(self.value_separator_pairs.is_empty() && self.final_value_opt.is_none()) {
```

Reply:
That line is so that if the block is completely empty we do nothing, if we have something we don't want to execute that block

Reply:
OH, i see it now
I'm the dumb